mis_builder/tests/__init__.py

```diff
@@ -6,6 +6,7 @@
 from . import test_multi_company_aep
 from . import test_aggregate
 from . import test_data_sources
+from . import fake_models
```

Comment:
@marielejeune The fake model should not be imported otherwise it will pollute the current registry.

Reply:
Obviously ... Thanks :-)